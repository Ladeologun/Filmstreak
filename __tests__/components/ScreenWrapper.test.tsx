import renderer from 'react-test-renderer';
import {test,expect} from '@jest/globals';
import { ScreenWrapper } from "~components";


test('renders correctly', () => {
    const tree = renderer.create(<ScreenWrapper hideStatusBar={false} />).toJSON();
    expect(tree).toMatchSnapshot();
});
