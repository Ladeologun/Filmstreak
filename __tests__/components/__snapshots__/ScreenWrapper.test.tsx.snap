// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  style={
    [
      {
        "backgroundColor": "#0F111D",
        "flex": 1,
        "paddingHorizontal": 25,
        "paddingTop": 0,
      },
      undefined,
    ]
  }
/>
`;
